themes/root/templates/Helpers/icons/svg-sprite.phtml

```diff
@@ -1,3 +1,3 @@
 <svg class="icon--svg"<?=$this->attrs ?> aria-hidden="true">
-    <use xlink:href="<?=$this->imageLink($src) ?>#<?=$this->escapeHtmlAttr($this->icon) ?>"></use>
+    <use xlink:href="<?=$this->imageLink($src) ?>#<?=$this->icon ?>"></use>
```

Comment:
Was the removal of escaping here intentional? Is proper escaping guaranteed elsewhere?